README.md

```diff
@@ -20,5 +20,97 @@ FLARE is currently a pure Python framework which doesn't need build. Several dep
 
 That's it. Enjoy training agents with FLARE!
 
-## Quick example
-*TODO*
+## A quick tutorial in 5 minutes
+Some quick examples can be found in `flare/examples`. Below we show step-by-step code to train the "CartPole-v0" game from OpenAI Gym.
+
+1. First we import different Python modules which all together constitute the overall training pipeline. Some modules such as ```SimpleAC```, ```SimpleModelAC```, and ```SimpleRLAgent``` were already implemented by us as *examples* of Algorithm, Model, and Agent classes. The users are encouraged to modify or write their own for more complex problems. This advanced topic of user customizing modules is explained in details in [Customize Your Own Modules](#).
+
+```python
+"""
+flare/examples/ac_example.py
+"""
+
+## import PyTorch nn module
+import torch.nn as nn
+## from the FLARE algorithm zoo import a simple Actor-critic algorithm
+from flare.algorithm_zoo.simple_algorithms import SimpleAC
+## import the Manager to manage agents running in parallel
+from flare.framework.manager import Manager
+## from the FLARE model zoo import a simple model for AC
+from flare.model_zoo.simple_models import SimpleModelAC
+## from the FLARE agent zoo import a simple RL agent
+from flare.agent_zoo.simple_rl_agents import SimpleRLAgent
+## the agent collects data in an on-line manner
+from flare.framework.agent import OnPolicyHelper
+## import a wrapped version of OpenAI gym
+from flare.framework.env import GymEnv
+```
+
+2. Then we configure the problem and initialize the environments.
+
+```python
+game = "CartPole-v0"
+
+num_agents = 16    ## 16 agents running in parallel
+num_games = 8000   ## each agent runs for 8000 games
+
+envs = []
+for _ in range(num_agents):
+    envs.append(GymEnv(game))  ## init an environment for each agent
+
+## get the environment observation dimension and the number of actions
+state_shape = envs[-1].observation_space.shape[0]
+num_actions = envs[-1].action_space.n
+```
+
+3. Next we construct the neural network and choose the algorithm to train the network.
+
+```python
+## For this simple problem, our model uses an MLP to directly map
+## each input vector to a latent embedding of 128 dimensions.
```

Comment:
We first create a `Model` object, `SimpleModelAC`, and define the network structure for it. For this simple problem, our model uses an MLP to directly map each input vector to a latent embedding of 128 dimensions.